build/dependencies.yaml

```diff
@@ -99,6 +101,20 @@ dependencies:
     - path: cluster/images/etcd-empty-dir-cleanup/Dockerfile
       match: us\.gcr\.io\/k8s-artifacts-prod\/build-image\/debian-base:v\d+\.\d+\.\d+
 
+  - name: "k8s.gcr.io/debian-hyperkube-base"
+    version: 1.0.0
+    refPaths:
+    - path: build/debian-hyperkube-base/Makefile
+      match: TAG\?=
+
+  - name: "k8s.gcr.io/debian-hyperkube-base: dependents"
+    version: 0.12.1
```

Comment:
does this not match the `TAG=0.12.2` in the Makefile because we never released 0.12.2?

Reply:
Yep, that image appear to have never been built or pushed.